pkg/webhooks/regularuser/regularuser.go

```diff
@@ -266,8 +271,9 @@ func isNetNamespaceValid(s *RegularuserWebhook, request admissionctl.Request) bo
 		return false
 	}
 	// Check if the name is bad or is privileged
-	if namespace.PrivilegedNamespaceRe.Match([]byte(netNamespace.Name)) ||
-		namespace.BadNamespaceRe.Match([]byte(netNamespace.Name)) {
+	if netNamespaceRE.Match([]byte(netNamespace.Name)) ||
+		namespace.BadNamespaceRe.Match([]byte(netNamespace.Name)) ||
+		hookconfig.IsPrivilegedNamespace(netNamespace.Name) {
```

Comment:
Are net namespaces privileged? It's strange to have the comment say check if X or Y and then have three things being checked. These can be "annotated" in this way, it seems:

```go
	// Check if the name is bad or is privileged
	if netNamespaceRE.Match([]byte(netNamespace.Name)) || // privileged
		namespace.BadNamespaceRe.Match([]byte(netNamespace.Name)) || // "bad" (domain names suffixes)
		hookconfig.IsPrivilegedNamespace(netNamespace.Name) { // privileged
		return false
	}
```

This is as if the comment reads: Check if X or Y or X. I'd like to see the idea of privileged namespaces consolidated into the one single function since if it's privileged _here_, it should be considered privileged _everywhere_ without needing this X or Y or X construct.

Reply:
Now that this conditional is back to a single `OR`, do you think it's still necessary to add the annotation to each line?